maskrcnn_benchmark/modeling/make_layers.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+"""
+Miscellaneous utility functions
+"""
+
+import torch
+from torch import nn
+from torch.nn import functional as F
+from maskrcnn_benchmark.config import cfg
+from maskrcnn_benchmark.layers import Conv2d
+from maskrcnn_benchmark.modeling.poolers import Pooler
+
+
+def get_group_gn(dim, dim_per_gp, num_groups):
+    """get number of groups used by GroupNorm, based on number of channels."""
+    assert dim_per_gp == -1 or num_groups == -1, \
+        "GroupNorm: can only specify G or C/G."
+
+    if dim_per_gp > 0:
+        assert dim % dim_per_gp == 0, \
+            "dim: {}, dim_per_gp: {}".format(dim, dim_per_gp)
+        group_gn = dim // dim_per_gp
+    else:
+        assert dim % num_groups == 0, \
+            "dim: {}, num_groups: {}".format(dim, num_groups)
+        group_gn = num_groups
+
+    return group_gn
+
+
+def _group_norm():
+    def gn_layer_from_cfg(out_channels, affine=True, divisor=1):
+        out_channels = out_channels // divisor
+        dim_per_gp = cfg.MODEL.GROUP_NORM.DIM_PER_GP // divisor
+        num_groups = cfg.MODEL.GROUP_NORM.NUM_GROUPS // divisor
+        eps = cfg.MODEL.GROUP_NORM.EPSILON # default: 1e-5
+        return torch.nn.GroupNorm(
+            get_group_gn(out_channels, dim_per_gp, num_groups), 
+            out_channels, 
+            eps, 
+            affine
+        )
+    return gn_layer_from_cfg
+group_norm = _group_norm()
```

Comment:
we can probably remove this extra indirection, as we don't use `_group_norm` anywhere else?
By that, I mean making `group_norm` be simply `gn_layer_from_cfg`